src/AdvApi32.Desktop/AdvApi32.cs

```diff
@@ -888,14 +933,169 @@ public static partial class AdvApi32
+
+        /// <summary>
+        /// The GetSecurityInfo function retrieves a copy of the security descriptor for an object specified by a handle.
+        /// </summary>
+        /// <param name="handle">A handle to the object from which to retrieve security information.</param>
+        /// <param name="ObjectType">SE_OBJECT_TYPE enumeration value that indicates the type of object.</param>
+        /// <param name="SecurityInfo">A set of bit flags that indicate the type of security information to retrieve. This parameter can be a combination of the <see cref="SECURITY_INFORMATION"/> bit flags.</param>
+        /// <param name="ppsidOwner">A pointer to a variable that receives a pointer to the owner SID in the security descriptor returned in ppSecurityDescriptor or NULL if the security descriptor has no owner SID. The returned pointer is valid only if you set the OWNER_SECURITY_INFORMATION flag. Also, this parameter can be NULL if you do not need the owner SID.</param>
+        /// <param name="ppsidGroup">A pointer to a variable that receives a pointer to the primary group SID in the returned security descriptor or NULL if the security descriptor has no group SID. The returned pointer is valid only if you set the GROUP_SECURITY_INFORMATION flag. Also, this parameter can be NULL if you do not need the group SID.</param>
+        /// <param name="ppDacl">A pointer to a variable that receives a pointer to the DACL in the returned security descriptor or NULL if the security descriptor has no DACL. The returned pointer is valid only if you set the DACL_SECURITY_INFORMATION flag. Also, this parameter can be NULL if you do not need the DACL.</param>
+        /// <param name="ppSacl">A pointer to a variable that receives a pointer to the SACL in the returned security descriptor or NULL if the security descriptor has no SACL. The returned pointer is valid only if you set the SACL_SECURITY_INFORMATION flag. Also, this parameter can be NULL if you do not need the SACL.</param>
+        /// <param name="ppSecurityDescriptor">
+        /// A pointer to a variable that receives a pointer to the security descriptor of the object. When you have finished using the pointer, free the returned buffer by calling the <see cref="LocalFree(void*)"/> function.
+        /// This parameter is required if any one of the <paramref name="ppsidOwner"/>, <paramref name="ppsidGroup"/>, <paramref name="ppDacl"/>, or <paramref name="ppSacl"/> parameters is not NULL.
+        /// </param>
+        /// <returns>If the function succeeds, the return value is <see cref="Win32ErrorCode.ERROR_SUCCESS"/>, otherwise it return one of the default error codes.</returns>
+        [DllImport(nameof(AdvApi32), SetLastError = true)]
+        public static unsafe extern uint GetSecurityInfo(
+            IntPtr handle,
+            SE_OBJECT_TYPE ObjectType,
+            SECURITY_INFORMATION SecurityInfo,
+            IntPtr ppsidOwner,
+            IntPtr ppsidGroup,
```

Comment:
Given the docs for these two parameters indicates they are 'a pointer to a pointer', I think we should make these `ref void*` to make calling this method much easier. And if we can type it more precisely than `void*`, so much the better.